diff --git a/src/index.js b/src/index.js
index dc295cc311fc6fcbee29c5a421fa5fae6bd5ffaa..29370f9e135569b21118fbde2eee00ba70a76536 100644
--- a/src/index.js
+++ b/src/index.js
@@ -420,10 +420,10 @@ export const publish = async (options) => {
           },
         )
       }
-      if (existsSync(path.resolve(rootDir, 'pnpm-lock.yaml'))) {
-        console.info('  Updating pnpm-lock.yaml...')
+      if (existsSync(path.resolve(rootDir, 'bun.lock'))) {
+        console.info('  Updating bun.lock...')
         try {
-          execSync('pnpm install --no-frozen-lockfile')
+          execSync('bun install --no-frozen-lockfile')
         } catch (/** @type {any} */ err) {
           throw new Error(err.stdout.toString())
         }
@@ -457,7 +457,7 @@ export const publish = async (options) => {
   for (const pkg of changedPackages) {
     const packageDir = path.join(rootDir, pkg.packageDir)
 
-    const cmd = `cd ${packageDir} && pnpm publish --tag ${npmTag} --access=public --no-git-checks`
+    const cmd = `cd ${packageDir} && bun publish --tag ${npmTag} --access=public --no-git-checks`
     console.info(`  Publishing ${pkg.name}@${version} to npm...`)
     execSync(cmd, {
       stdio: [process.stdin, process.stdout, process.stderr],
